src/Compilers/CSharp/Portable/Binder/Binder_Patterns.cs

```diff
@@ -2,14 +2,16 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
-using Microsoft.CodeAnalysis.CSharp.Symbols;
-using Microsoft.CodeAnalysis.CSharp.Syntax;
-using Roslyn.Utilities;
 using System.Collections.Generic;
 using System.Collections.Immutable;
 using System.Diagnostics;
-using System;
+using System.Diagnostics.CodeAnalysis;
+using Microsoft.CodeAnalysis.CSharp.Symbols;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
 using Microsoft.CodeAnalysis.PooledObjects;
+using Roslyn.Utilities;
+
+#nullable enable
```

Comment:
nit: while I appreciate the sentiment, this PR is large enough that throwing nullability changes in doesn't help :-(

Reply:
Sorry; it was helpful during development.

---
In reply to: [380880371](https://github.com/dotnet/roslyn/pull/41503#discussion_r380880371) [](ancestors = 380880371)